website/contribute/developer-setup.md

```diff
@@ -275,6 +275,18 @@ Discussion about contributing code to Hudi happens on the [dev@ mailing list](/c
 
 This Quick Video will give a code walkthrough to start with [watch](https://www.youtube.com/watch?v=N2eDfU_rQ_U).
 
+## Running unit tests and local debugger via Intellij IDE
+
+0. Build the project with the intended profiles via the `mvn` cli, for example for spark 3.2 use `mvn clean package -Dspark3.2 -Dscala-2.12 -DskipTests`. 
+1. Install the "Maven Helper" plugin from the Intellij IDE.
+2. Make sure IDEA uses Maven to build/run tests:
+   * You need to select the intended Maven profiles (using Maven tool pane in IDEA): select profiles you are targeting for example `spark2.4` and `scala-2.11` or `spark3.2`, `scala-2.12` etc. 
+   * Add `.mvn/maven.config` file at the root of the repo w/ the the profiles you selected in the pane: `-Dspark3.2` `-Dscala-2.12`
+3. Make sure you change (temporarily) the `scala.binary.version` in the root `pom.xml` to the intended scala profile version. For example if running with spark3 `scala.binary.version` should be `2.12`
+4. In the root pom.xml under `maven-surefire-plugin`, under this plugin's `<configuration>`  make sure to add `<skipTests>false<skipTests>`
```

Comment:
I don't think this is necessary

Reply:
I think this is necessary for certain tests, when I tried running `TestMORDataSource` from IDE it would skip the test. Only when doing step 4 was it working for me. If you are able to run these tests without doing the step 4 i can remove it.

Reply:
We should ideally avoid any local changes to POM as they inevitable will be getting committed occasionally

Reply:
BTW, i just tried `TestMORDataSource` and it works fine for me w/o `skipTests` flag

Reply:
> BTW, i just tried `TestMORDataSource` and it works fine for me w/o `skipTests` flag
got it, will remove `4.`